packages/react-vapor/src/components/collapsible/CollapsibleContainerConnected.tsx

```diff
@@ -72,7 +72,7 @@ export const CollapsibleContainerDisconnected: React.FunctionComponent<
                 </div>
             }
             expandedOnMount={expandedOnMount}
-            headerClasses={classNames(styles.header, expanded ? 'bg-light-grey' : 'bg-white')}
+            headerClasses={classNames(styles.header)}

```

Comment:
The  3 first `CollapsibleConnected` in the demo were grey because of an added className but it's white by default from what i've seen in the admin. This is the only place where we use the className tag and no color either so I think the demo can stay white :eyes: 
![image](https://user-images.githubusercontent.com/63734941/106814435-61390900-6640-11eb-8f05-5d7c3b7c57b5.png)

The collapsible containers ones are fixed :+1: